### ISSUE TITLE
Layer [shortcut] not implemented

### ISSUE THREAD

**Issue by Urviagrawal** [pjreddie/darknet#574]
@pjreddie I am getting this error while using yolov3 cfg file. 
Parsing ./cfg/yolov3.cfg
Layer [shortcut] not implemented
This is the error

**Comment by abagshaw** [comment]
@Urviagrawal Are you trying to use YOLOv3 with https://github.com/thtrieu/darkflow or some other darknet port? YOLOv3 is relatively new and is not supported (yet) by darkflow and probably by most other darknet ports.

**Comment by nemkin** [comment]
Hello!

I have the same problem.

I have checked out from this remote:
https://github.com/thtrieu/darkflow.git

On commit:
b2aee0000cd2a956b9f1de6dbfef94d53158b7d8

Ran:
pip install . 

Downloaded the following cfg and model:
https://github.com/pjreddie/darknet/blob/master/cfg/yolov3.cfg
https://pjreddie.com/media/files/yolov3.weights 

Ran:
flow --model cfg/yolov3-new.cfg --load bin/yolov3.weights 

Got the following error:

Parsing ./cfg/yolov3.cfg
Layer [shortcut] not implemented

**Comment by abagshaw** [comment]
Yes, for now you'll have to use darknet directly. Stay tuned to this issue: https://github.com/thtrieu/darkflow/issues/665 regarding YOLOv3 support on darkflow.

**Comment by Urviagrawal** [comment]
@abagshaw thank you. Can you help me in using darknet using python?

**Comment by fbadaud** [comment]
hello , I am trying to use create .pb file for a YOLOV3  already trained network . For this I was using darkflow and it works perfectly  on YOLOV2 but for V3 it looks that some definitions are mot existing already.  
Is there another method to create the pb files for YOLOV3 ?
 thanks in advance for your support

**Comment by fionasoon1** [comment]
Try this and it works
python flow --model cfg/yolo.cfg --load bin/yolov2.weights --demo video.mp4 --saveVideo

but not work for yolov3

**Comment by anshulg03** [comment]
Guys how to convert yolov3 weights to pb file?

**Comment by jeffreyhong0217** [comment]
How to convert yolov3 weights to pb file?Anybody knows???

**Comment by AlexeyAB** [comment]
@jeffreyhong0217 
Read this, how to convert `cfg/weights` to `.ckpt/pb/meta` https://github.com/AlexeyAB/darknet#yolo-v3-in-other-frameworks

**Comment by jeffreyhong0217** [comment]
@AlexeyAB OK!Thanks!

**Comment by angyee** [comment]
I am trying to convert yolov3_spp to caffemodel. please help using https://github.com/hojel/caffe-yolo-model this repo I am trying but 

getting error

$ sudo python3 create_yolo_prototxt.py yolov3-spp.cfg yolov3-spp.prototxt
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:shortcut layer is not supported
ERROR:root:route layer is not supported
ERROR:root:route layer is not supported
ERROR:root:route layer is not supported
ERROR:root:yolo layer is not supported
ERROR:root:route layer is not supported
ERROR:root:upsample layer is not supported
ERROR:root:route layer is not supported
ERROR:root:yolo layer is not supported
ERROR:root:route layer is not supported
ERROR:root:upsample layer is not supported
ERROR:root:route layer is not supported
ERROR:root:yolo layer is not supported
